BookPlayer/Library/LibraryViewController.swift

```diff
@@ -120,6 +120,20 @@ class LibraryViewController: BaseListViewController, UIGestureRecognizerDelegate
     }
 
     func handleDelete(playlist: Playlist, indexPath: IndexPath) {
+        if playlist.books?.count == 0 {
+            self.library.removeFromItems(playlist)
+
+            DataManager.saveContext()
```

Comment:
I've seen that we use this sequence many times
`saveContext`
`beginUpdates`
`deleteRows`
`endUpdates`
We might want to put that in its own function that accepts an array of IndexPaths 🤔
Thoughts?

Reply:
I guess you're right. I just didn't want to introduce too many new things in this, but it makes sense.

Reply:
After some thinking I'm no longer convinced we would win that much by creating an helper here. I think it will make that part more complex for brevities sake and if it comes to that I prefer the more easily readable solution.